synchro/__init__.py

```diff
@@ -798,3 +801,24 @@ def __exit__(self, *args):
 
     def get_keys(self):
         return Cursor(self.synchronize(self.delegate.get_keys)())
+
+    def create_encrypted_collection(
+        self,
+        database,
+        name,
+        encrypted_fields,
+        kms_provider=None,
+        master_key=None,
+        **kwargs,
+    ):
+        coll, ef = self.synchronize(self.delegate.create_encrypted_collection)(
+            database=database.delegate,
+            name=name,
+            encrypted_fields=encrypted_fields,
+            kms_provider=kms_provider,
+            master_key=master_key,
+            **kwargs,
+        )
+        client = MongoClient(delegate=coll.database.client)
+        database = Database(client, coll.database.name)
```

Comment:
No need to recreate the client and database here, let's reuse the `database` param.

Reply:
Done.